### DISCUSSION TITLE
Bulk Delete Custom Confirmation in Bootstrap

### DISCUSSION THREAD

**Discussion by abhaypithadiya** [Power-Components/livewire-powergrid#603] [Q&A] [answered]
How can I get a custom confirmation when using bulk delete? Not browser alert. Any Ideas?

**Comment by dansysanalyst** [comment, marked as answer]
<img width="1367" alt="CleanShot 2022-09-02 at 13 10 44@2x" src="https://user-images.githubusercontent.com/79267265/188127333-e46f928d-4b59-492a-9c1b-23ef2cd3aff8.png">

👀 https://github.com/Power-Components/powergrid-demo/blob/main/app/Http/Livewire/DishesTable.php
👀 https://github.com/Power-Components/powergrid-demo/blob/main/app/Http/Livewire/DeleteDish.php

👉 https://github.com/Power-Components/powergrid-demo

**Comment by abhaypithadiya** [comment]
Hey! I tried this. I changed the classes to Bootstrap once but doesn't work. It doesn't display the popup. Could you please help?
